src/App/App.js

```diff
@@ -0,0 +1,83 @@
+import './App.css';
+import React, { Component } from 'react';
+import Header from '../Header/Header';
+import CardContainer from '../CardContainer/CardContainer';
+import MovieDisplay from '../MovieDisplay/MovieDisplay';
+// import MovieDetails from './MovieDetails/MovieDetails';
+// import movieData from '../data/dummyData';
+import { fetchData } from '../util.js'
+
+
+class App extends Component {
+ constructor(){
+   super();
+   this.state = {
+     movies: [],
+     currentSelectionId: null,
+     error: null,
```

Comment:
I think this is a great idea to have any error messages held in state. We will be able to easily populate the inner text of an element in our error display component to give the user valuable info.